nomad/local/grapl-local-infra.nomad

```diff
@@ -65,6 +65,18 @@ variable "PLUGIN_WORK_QUEUE_DB_PASSWORD" {
   default     = "postgres"
 }
 
+variable "ORGANIZATION_MANAGEMENT_DB_USERNAME" {
```

Comment:
Not really a comment about this PR _per se_, but is there an important distinction between all-uppercase variables and all-lowercase variables? I just noticed these differ from the ones in `grapl-core.nomad` and `integration-tests.nomad` in this respect.

Reply:
Not that I'm aware of, I tried to follow existing implementations which used all uppercase, but we can change this if needed. Leaving for now.

Reply:
I'm not particularly sure, I followed standards set by other services, this is a good thing to consider though. Leaving for now but would be happy to refactor this service along with others to have consistent casing if that makes more sense. Leaving for now.